config/locales/cy/brexit_landing_page.yml

```diff
@@ -15,14 +15,14 @@ cy:
         list_block: |
           Gwiriwch beth sydd angen i chi wneud os ydych yn:
           <ul>
-            <li><strong><a href="/prepare-import-to-uk-after-brexit" data-ecommerce-row="true" data-ecommerce-path="/prepare-import-to-uk-after-brexit">mewnforio nwyddau o’r UE</a></strong></li>
-            <li><strong><a href="/prepare-export-from-uk-after-brexit" data-ecommerce-row="true" data-ecommerce-path="/prepare-export-from-uk-after-brexit">allforio nwyddau i’r UE</a></strong></li>
-            <li><strong><a href="/transport-goods-from-uk-by-road" data-ecommerce-row="true" data-ecommerce-path="/prepare-export-from-uk-after-brexit">cludo nwyddau allan o’r DU ar y ffyrdd</a></strong></li>
+            <li><strong><a href="/prepare-import-to-uk-after-brexit" data-module="track-click" data-track-category="transition-landing-page" data-track-action="/prepare-import-to-uk-after-brexit" data-track-label="mewnforio nwyddau o’r UE">mewnforio nwyddau o’r UE</a></strong></li>
```

Comment:
Instead of having `data-module=track-click` on every link, we can add it to a wrapping container and the tracking should still work. Perhaps something in the buckets view?

Reply:
amended! and put it here: https://github.com/alphagov/collections/pull/1448/files#diff-750d5b8c00478873657c9c932e94bf6fR27

Reply:
It may be better placed inside the buckets.html.erb file, just so it's obvious when you look at the file, rather than it being in the more general page view - but both should work ok.

Reply:
I've confused myself, isn't that where it is currently?
Line 27 of _buckets.html.erb?

Reply:
Yes, sorry 🤦‍♀ Ignore me!